docs/workflow_solutions/creating_sandbox_apps.md

```diff
@@ -0,0 +1,178 @@
+
+    ![Form for new Sandbox App](images/sandbox_new_app_form.png)
+
+    We will add the following values in the form:
+    - **Directory Name:** FSL
+    - **Git Remote:** `https://gitlab.rc.uab.edu/rc/bc_uab_igv.git`
+    After completing the above steps, then click "**Submit**"
+
+1. When this is done successfully, you should see the below screen also showing your new FSL Sandbox App.
+    ![New Sandbox App](images/new_fsl_sandbox_app.png)
+
+    <!-- markdownlint-disable MD046 -->
+    !!! note
+        You can also check the option "Create a new Git Project from this?", about creating a new git repo, if you want to use it to maintain your own instance of the app. In this tutorial, we will not be selecting this option.
+    <!-- markdownlint-enable MD046 -->
+
+1. Click on "Files" to take a look at the app files (local git repo working directory) you just cloned. We will modify some of the files here, because its contents are not relevant to FSL.
+
+    ![File list for Sandbox App](images/file_sandbox_app.png)
+
+    1. We first modify "manifest.yml" file (highlight the file, and click on edit) to change metadata for the sandbox app. Change the variables appropriately. In this case we will change the following:
+
+        ![Metadata contained in manifest.yml file](images/manifest_yml_sandbox.png)
+
+    1. Next, select "form.yml.erb" and click on "Edit" . Replace the content of this file with the .yml file below.
+
+        ``` yaml
+
+            <%-
+        partitions = OodAppkit.clusters[:SLURM_CLUSTER].custom_config[:partitions]
+        -%>
+        ---
+        cluster: "SLURM_CLUSTER"
+        attributes:
+        bc_num_hours:
+            value: 1
+
+        bc_num_slots:
+            widget: "number_field"
+            label: Number of CPU
+            value: 1
+            min: 1
+            max: 24
+            step: 1
+
+        bc_num_mems:
+            widget: "number_field"
+            label: Memory per CPU (GB)
+            value: 4
+            min: 1
+            max: 128
+            step: 1
+
+        bc_num_gpus:
+            widget: "number_field"
+            label: Number of GPUs
+            value: 1
+            min: 0
+            max: 4
+            step: 1
+
+        bc_partition:
+            widget: select
+            label: Partition
+            options:
+            <%- partitions.each do |p| -%>
+            - [ "<%= p %>", "<%= p %>" ]
+            <%- end -%>
+
+        version:
+            widget: select
+            label: "FSL version"
+            help: "This defines the version of FSL you want to load."
+            options:
+            - [ "6.0.3", "FSL/6.0.3" ]
+
+        form:
+        - version
+        - bc_num_hours
+        - bc_partition
+        - bc_num_gpus
+        - bc_num_slots
+        - bc_num_mems
+        - bc_email_on_started
+
+        ```
+
+    Here we are specifying what the form for the app is going to look like.
```

Comment:
Move this and the admonition up to the top of this numbered item, helps to draw attention to them and make them feel like part of the item.